clients/edgex-foundry/device_client.go

```diff
@@ -236,7 +227,7 @@ func (efc *EdgexDeviceClient) UpdatePropertyState(ctx context.Context, propertyN
 		dps.PutURL = putURL
 	}
 	// set the device property to desired state
-	efc.V(5).Info("setting the property to desired value", "property", dps.Name)
+	klog.V(5).Infof("setting the property %s to desired value", dps.Name)
```

Comment:
Adding device name into the log info may be better.

Reply:
The device name has been added.